generator/generator.go

```diff
@@ -77,7 +77,12 @@ func NewGenerator() *Generator {
 	g.t.Funcs(funcs)
 
 	for _, asset := range assets.AssetNames() {
-		g.t = template.Must(g.t.Parse(string(assets.MustAsset(asset))))
+		a, err := assets.Asset(asset)
```

Comment:
Any reason for the change here?

Reply:
Otherwise the build fails with a fresh installation of master. I think it's a change in a newer version of go-bindata?

Reply:
I dug a bit more into this, and I have a slightly better idea of what is going on here. It's not a problem with a *newer* version; it's a problem with an *older* version. `MustAsset` was added in [this commit](https://github.com/go-bindata/go-bindata/commit/81e20ed1b787aee59470382444baf4cb6f9ae6f0#diff-e159afc50edd71481df34fac359f9be6) to go-bindata/go-bindata. This showed up in go-bindata/go-bindata v3.1.2. I'm specifically saying go-bindata/go-bindata because the `install-deps` make target installs `github.com/jteeuwen/go-bindata/go-bindata`. I'm not entirely sure of the history there, but it appears that the original owner of that repository disappeared. The repository currently at that location is a hard copy of it at a certain point in time. (Amusingly, the git tag for this point in time is v3.0.7, but the version number defined in version.go is v3.1.0, which almost made me tear out my hair.)

Thus, the version of go-bindata pointed to by the Makefile does not generate the `MustAsset` function. So, my branch choked on it, and I changed it.

I think there are a couple options here:

1. Take this change as-is, which is backward-compatible with the older version of go-bindata specified in `install-deps`
2. I can change the Makefile (and go.mod, go.sum etc) to point to go-bindata/go-bindata and regenerate, and then revert my change in generator.go.

I think I'd prefer option 2. If nothing else, pointing to a version of go-bindata that is in active development is a good enough reason to do it. Let me know what you think!

Reply:
Yeah, I don't touch this repo all that often, so I'm not surprised that it doesn't get much dependency love. 

I've personally moved to a better way of managing dependencies in other project, but I won't make you tackle that change.  For now, I think I'd rather you use `https://github.com/kevinburke/go-bindata` since it's the one installed from homebrew, and is just as actively developed as the other one.

I'll change it to a more repeatable build at a later date.
Sorry about that, and thanks for the contribution!